provider/go.mod

```diff
@@ -19,6 +19,9 @@ replace (
 	github.com/hashicorp/terraform-provider-aws => ../upstream
 	github.com/hashicorp/terraform-provider-aws/shim => ./shim
 	github.com/hashicorp/vault => github.com/hashicorp/vault v1.2.0
+
+	github.com/pulumi/pulumi-terraform-bridge/pf => github.com/pulumi/pulumi-terraform-bridge/pf v0.14.2-0.20230802145745-14057592edb9
+	github.com/pulumi/pulumi-terraform-bridge/v3 => github.com/pulumi/pulumi-terraform-bridge/v3 v3.55.1-0.20230802145745-14057592edb9
```

Comment:
Can we set these versions as a `require` instead of as a `replace`. That way it will be updated normally after merging.